helpers/deepl/common.js

```diff
@@ -0,0 +1 @@
+export const JA = "ja";
```

Comment:
minor: You can place this variable in `ROOT/constants/deepl.js`.

Reply:
I understand.
Changed the place of the variable. -->[link](https://github.com/yuki-o1o5/hacker-news-in-japanese/commit/a75e264c6b0b1ea84b1ac44fcad2b28a8ef04502)